src/test/kotlin/pt/up/fe/ni/website/backend/controller/EventControllerTest.kt

```diff
@@ -271,6 +274,89 @@ internal class EventControllerTest @Autowired constructor(
         }
     }
 
+    @NestedTest
+    @DisplayName("GET events/category/{category}")
```

Comment:
Was this not tested before? I'm a bit confused since it seems out of scope for this PR

Reply:
I honestly don't remember this part, my guess is that I updated the branch and needed to modify it to include the gallery.